setup.cfg

```diff
@@ -0,0 +1,4 @@
+[nosetests]
+tests = test/unit,
+        utils,
+        roles/openshift_master_facts/test,
```

Comment:
Possibly not. I moved the Python files under `test/` just to have a better organization and reuse the `test/` top-level dir.

@detiber, I would even consider moving `test/unit/modify_yaml_tests.py` -> `library/test_modify_yaml.py`, so that test files live close to what they test.